clients/repo_client.go

```diff
@@ -42,3 +45,8 @@ type RepoClient interface {
 	Search(request SearchRequest) (SearchResponse, error)
 	Close() error
 }
+
+// PackageManagerClient interface is used by Scorecard checks to access a package manager.
+type PackageManagerClient interface {
+	Get(URI string, packagename string) (*http.Response, error)
+}
```

Comment:
Since this interface is only used in the `cmd` package, let's define this as an *unexposed* interface there?

Reply:
The reason I decided to have the interface in the `clients` package is to be consistent with other `clients`. If we don't expose it can't be consumed as a parameter in the `cmd` package.

Reply:
Agreed, moved it to the `cmd` package and changed the scope of the interface to be not public. PTAL.